commons-math-transform/src/test/java/org/apache/commons/math4/transform/TransformUtilsTest.java

```diff
@@ -22,27 +22,34 @@
 import org.junit.Test;
 
 import org.apache.commons.math3.analysis.function.Sin;
+import org.junit.jupiter.api.function.Executable;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 /**
  * Tests for {@link TransformUtils}.
  */
 public class TransformUtilsTest {
+
     private static final Sin SIN_FUNCTION = new Sin();
     private static final DoubleUnaryOperator SIN = x -> SIN_FUNCTION.value(x);
 
-    @Test(expected = TransformException.class)
+    @Test
     public void testSampleWrongBounds() {
-        TransformUtils.sample(SIN, Math.PI, 0.0, 10);
+        final Executable testMethod = () -> TransformUtils.sample(SIN, Math.PI, 0.0, 10);
```

Comment:
I would remove the import of `Executable` and just use the lambda function direct within assertThrows.

Reply:
happy to refactor to your style, i just find it easier this way with junior/new developers than making messy lambda function blocks.